UPGRADE-3.1.md

```diff
@@ -107,7 +107,7 @@ FrameworkBundle
    framework:
        cache:
            pools:
-               serializer:
+               cache.serializer:
```

Comment:
I understand that we want the service to be `cache.serializer`. But imo we should then add the `cache.` prefix here: https://github.com/nicolas-grekas/symfony/blob/cache-config/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L1059

Otherwise, the full config path we use here is `framework.cache.pools.cache.serializer` which imo is not really clear why I should use the `cache` part twice here.